client-tools/Shell/RequestFuzzer.h

```diff
@@ -0,0 +1,123 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2022 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Julia Puget
+////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include <charconv>
+#include <deque>
+#include <map>
+#include <random>
+#include <string>
+#include <unordered_map>
+
+#include <optional>
+
+namespace arangodb {
+namespace fuzzer {
+
+enum CharOperation { ADD_STRING, ADD_INT_32, MAX_CHAR_OP_VALUE };
+
+enum LineOperation {
+
+  COPY_LINE,
```

Comment:
Please follow ArangoDB naming convention mentioned in our coding guidelines.

Reply:
from what I've seen in the code guidelines, this enum name is ok, isn't it?

Reply:
No, should be `kCopyLine`, `kInjectRandByteInLine` etc.

Reply:
Addressed